portability-transfer/src/main/java/org/dataportabilityproject/transfer/PortabilityInMemoryDataCopier.java

```diff
@@ -86,31 +90,45 @@ private void copyHelper(
       UUID jobId,
       AuthData exportAuthData,
       AuthData importAuthData,
-      Optional<ExportInformation> exportInformation)
-      throws IOException {
+      Optional<ExportInformation> exportInformation) {
 
     logger.debug("copy iteration: {}", COPY_ITERATION_COUNTER.incrementAndGet());
 
+    // TODO: read in retry strategies from a config, but that's for later in v1
+    RetryStrategy expBackoffStrategy = new ExponentialBackoffRetryStrategy(5, 10, 2);
+    RetryStrategyLibrary library = new RetryStrategyLibrary(new LinkedList<>(), expBackoffStrategy);
+
     // NOTE: order is important below, do the import of all the items, then do continuation
     // then do sub resources, this ensures all parents are populated before children get
     // processed.
     logger.debug("Starting export");
-    ExportResult<?> exportResult = export(jobId, exportAuthData, exportInformation);
-    logger.debug("Finished export");
-
-    if (exportResult.getType().equals(ResultType.ERROR)) {
-      logger.warn("Error happened during export: {}", exportResult.getMessage());
+    CallableExporter callableExporter = new CallableExporter(exporter, jobId, exportAuthData,
```

Comment:
nit: As far as this copier is concerned it wants a retrying exporter, retrying importer, etc. So I'd move this retrying logic into the CallableExporter class and call it RetryingExporter.

Reply:
Wouldn't that result in a lot of code duplication, since the same logic is used in CallableExporter and CallableImporter?

Reply:
you could refactor to a common place, just didn't think it needed to be a part of the copier, seems like it belongs more in the Callable*porter's

Reply:
seems like now there is some duplication in the copier, and it's stuff the copier doesn't need to know about